ragchain-structure/benchmark/README.md

```diff
@@ -25,36 +25,51 @@ The `Benchmark` module supports two types of evaluators: AutoEvaluator and Datas
 - [Qasper](https://allenai.org/data/qasper)
 - [MS-MARCO](./dataset-evaluator/ms-marco.md)
 - [Mr-Tydi](./dataset-evaluator/mr-tydi.md)
+- [Natural-question](./dataset-evaluator/natural-question.md)
+- [trivia-qa](./dataset-evaluator/trivia-qa.md)
+- [nfcorpus](./dataset-evaluator/nfcorpus.md)
+- [antique](./dataset-evaluator/antique.md)
+- [search-qa](./dataset-evaluator/search-qa.md)
+- [ASQA](./dataset-evaluator/asqa.md)
+- [DSTC11-track5](./dataset-evaluator/dstc11-track5.md)
+- [eli5](./dataset-evaluator/eli5.md)
 
 ## Supporting Metrics
 
-Each evaluator supports different metrics.
+Each evaluator supports various metrics. <br>
+The default metric refers to the metric that is essentially executed when you run the test file.<br>
```

Comment:
remove `when you run the test file`
The default metric refers to the metric that is essentially executed when you don't set `metrics` parameter.